ksp_plugin/plugin.cpp

```diff
@@ -658,17 +658,24 @@ void Plugin::SetPartApparentRigidMotion(
     PartId const part_id,
     RigidMotion<RigidPart, World> const& rigid_motion,
     DegreesOfFreedom<World> const& main_body_degrees_of_freedom) {
-  // Define |ApparentBubble| as the reference frame with the axes of
-  // |Barycentric| centred on the current main body.
+  // As a reference frame, |ApparentBubble| differs from |World| only by having
+  // the same axes as |Barycentric| and being nonrotating.  However, there is
+  // another semantic distinction: |Apparent| coordinates are uncorrected data
```

Comment:
Should this say `ApparentBubble` or is there another confusingly named frame?

Reply:
I mean `Apparent` as in the prefix, both `ApparentBubble` and `ApparentPileUp`.

Reply:
Maybe `Apparent...` then.